forms-flow-api/src/formsflow_api/models/form_process_mapper.py

```diff
@@ -214,16 +224,21 @@ def find_form_by_form_id(cls, form_id) -> FormProcessMapper:
         )  # pylint: disable=no-member
 
     @classmethod
+    @user_context
     def find_mapper_by_form_id_and_version(
-        cls, form_id: int, version: int
+        cls, form_id: int, version: int, **kwargs
     ) -> FormProcessMapper:
         """
         Return the form process mapper with given form_id and version.
 
         : form_id : form_id corresponding to the mapper
         : version : version corresponding to the mapper
         """
+        user: UserContext = kwargs["user"]
+        tenant_key: str = user.tenant_key
         query = cls.query.filter(
-            and_(cls.form_id == form_id, cls.version == version)
+            and_(
+                cls.form_id == form_id, cls.version == version, cls.tenant == tenant_key
```

Comment:
What would be the behavior in the default environment? when tenant_key is None it will match the mapper where the tenant is null right? just have a doubt here if that's the case

Reply:
Yes, it will match for None with None

Reply:
ok :thumbsup: